client/me/profile-gravatar/index.jsx

```diff
@@ -12,80 +11,36 @@ import { connect } from 'react-redux';
 import Animate from 'components/animate';
 import Gravatar from 'components/gravatar';
 import { recordGoogleEvent } from 'state/analytics/actions';
-import { isEnabled } from 'config';
```

Comment:
I think it's probably safer to keep the config flag, so we can flip this off quickly if something bad happens, then follow up by removing this in another PR. 

If you feel pretty confident here we can run with it, but maybe keep this PR bookmarked.